mgmtfn/k8splugin/kubeClient_test.go

```diff
@@ -367,8 +378,16 @@ func setupTestServer(c *C) {
 	tlsCfg := &tls.Config{
 		Certificates: []tls.Certificate{cert},
 		RootCAs:      caPool,
+		ClientAuth:   tls.VerifyClientCertIfGiven,
+		ClientCAs:    caPool,
 	}
 	tlsCfg.BuildNameToCertificate()
+	// Read the token
+	var contivK8Config ContivConfig
+
+	err = getConfig(testCfgFile, &contivK8Config)
+	token = contivK8Config.K8sToken
+
```

Comment:
That would work as well. Alternatively, you could have two config files and have the test code switch between those two. Either way, we need to have the test in place.